### PR TITLE
Oct

### PR DESCRIPTION
**Pull request by EngOmarAhmedIOS**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ Check ] All writings are my own.
- [ Check ] My entry follows the Codecademy Docs style guide.
- [ Check ] My changes generate no new warnings.
- [ Check ] I have performed a self-review of my own writing and code.
- [ Check ] I have checked my entry and corrected any misspellings.
- [ Check ] I have made corresponding changes to the documentation if needed.
- [ Check ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ Check ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ Check ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by EngOmarAhmedIOS** [comment]
can you please review my work ?

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+val2 = oct(0XAF) # Hexadecimal to octal  
+# Displaying result  
+# binary number  
+print("Octal value of 0b0101:",val)  
+# hexadecimal number  
+print("Octal value of 0XAF:",val2)  
+```
+### Output
+
+```py
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:69]:
````suggestion
```shell
````

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+# Calling function  
+val = oct(10)  
+# Displaying result  
+print("Octal value of 10:",val)  
+
+```
+
+### Output:
+
+```py
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:48]:
````suggestion
```shell
````

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@EngOmarAhmedIOS I did a first review and left a few comments.

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+[**This** method takes an argument and returns an integer converted into an octal string.]
+
+## Syntax
+
+```py
+oct(integer)
+```
+
+### Parameters
```
[the remark this answers is not in the document]
Comment (EngOmarAhmedIOS) [reply content/python/concepts/built-in-functions/terms/oct/oct.md:27]: you want me to delete the parameters header

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+```
+
+### Output:
+
+```py
+# Octal value of 10: 0o12
+```
+
+## Example 2
```
[the remark this answers is not in the document]
Comment (EngOmarAhmedIOS) [reply content/python/concepts/built-in-functions/terms/oct/oct.md:54]: I don't get what is the change I should do, I don't see any difference

[the remark this answers is not in the document]
Comment (SSwiniarski) [reply content/python/concepts/built-in-functions/terms/oct/oct.md:52]: @EngOmarAhmedIOS There's a line break after the header.

File: content/python/concepts/built-in-functions/terms/oct/oct.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (SSwiniarski) [reply content/python/concepts/built-in-functions/terms/oct/oct.md:26]: Yes

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
+---
+Title: 'oct()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+
+Description: 'Used to get an octal value of an integer number' # Required; ideally under 150 characters and starts with a present-tense verb (used in search engine results and content previews)
+
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
```
Comment (EngOmarAhmedIOS) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:7]:
```suggestion
  - 'Computer Science'
  - 'Data Science' 
  - 'Code Foundations'
```

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+Description: 'Used to get an octal value of an integer number' # Required; ideally under 150 characters and starts with a present-tense verb (used in search engine results and content previews)
+
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+  - 'Data Science' 
+  - 'Code Foundations'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Functions'
+  - 'Numbers'
```
Comment (EngOmarAhmedIOS) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:12]:
```suggestion
  - 'Numbers'
  - 'Data Types'
```

**Comment by EngOmarAhmedIOS** [comment]
I corrected all the requested changes. please review the final file @SSwiniarski

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
+---
+Title: 'oct()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:3]:
```suggestion
```
Remove extra line break.

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
+---
+Title: 'oct()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+
+Description: 'Used to get an octal value of an integer number' # Required; ideally under 150 characters and starts with a present-tense verb (used in search engine results and content previews)
+
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:5]:
```suggestion
```
Remove extra line break.

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+```
+Where `integer` is the value which is to be converted into an octal string.
+
+The function returns an octal string.
+
+## Example
+
+This example gets an octal value of a decimal number.
+
+
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:34]:
```suggestion
```
Remove extra line break.

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+```py
+# Python oct() function example  
+# Calling function  
+val = oct(10)  
+# Displaying result  
+print("Octal value of 10:",val)  
+
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:41]:
```suggestion
```
Remove extra line break.

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+print("Octal value of 0XAF:",val2)  
+```
+
+This example results in the following output:
+
+```shell
+# Octal value of 0b0101: 0o5
+# Octal value of 0XAF: 0o257
+```
+
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:72]:
```suggestion
```
Remove extra line break.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@EngOmarAhmedIOS I had a few more comments, all about removing extra line breaks in the entry.

**Review by SSwiniarski — APPROVED** [review]
@EngOmarAhmedIOS Looks good for a second review.

**Comment by EngOmarAhmedIOS** [comment]
Now I am officially contributor with CodeAcadmey Docs or Should I wait for a second review before celebrating this moment 😁 @SSwiniarski

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Functions'
+  - 'Numbers'
+  - 'Data Types'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **oct()** function takes an argument and returns an integer converted into an octal string.
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:17]:
```suggestion
The **`oct()`** function takes an integer and returns the corresponding octal string.

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+
+## Syntax
+
+```py
+oct(integer)
+```
+Where `integer` is the value which is to be converted into an octal string.
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:24]:
```suggestion
Where `integer` is the value to be converted into an octal string.

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+```shell
+# Octal value of 10: 0o12
+```
+
+## Example 2
+
+In Python, binary and hexadecimal values are integer values. So, the `oct()` function can return octal conversion of binary and hexadecimal values as well. 
+
+```py
+# Python oct() function example
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:51]:
```suggestion

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+```py
+oct(integer)
+```
+Where `integer` is the value which is to be converted into an octal string.
+
+The function returns an octal string.
+
+## Example
+
+This example gets an octal value of a decimal number.
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:30]:
```suggestion
This example returns the octal string for the value `10`.

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+
+In Python, binary and hexadecimal values are integer values. So, the `oct()` function can return octal conversion of binary and hexadecimal values as well. 
+
+```py
+# Python oct() function example  
+# Calling function  
+val  = oct(0b0101) # Binary to octal  
+val2 = oct(0XAF) # Hexadecimal to octal  
+# Displaying result  
+# binary number
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:56]:
```suggestion
# Binary number  

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+
+```py
+# Python oct() function example  
+# Calling function  
+val  = oct(0b0101) # Binary to octal  
+val2 = oct(0XAF) # Hexadecimal to octal  
+# Displaying result  
+# binary number  
+print("Octal value of 0b0101:",val)  
+# hexadecimal number
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:58]:
```suggestion
# Hexadecimal number  

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+
+This example results in the following output:
+
+```shell
+# Octal value of 10: 0o12
+```
+
+## Example 2
+
+In Python, binary and hexadecimal values are integer values. So, the `oct()` function can return octal conversion of binary and hexadecimal values as well.
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:48]:
```suggestion
In Python, binary and hexadecimal values are integer values. So, the `oct()` function can convert binary and hexadecimal values as well. 

```

**Review by caupolicandiaz — APPROVED** [review]
@EngOmarAhmedIOS @SSwiniarski Looks good. I just removed a couple of extra lines (there should never be multiple lines between content) and I decided to change the second example to a codebyte.

**Comment by EngOmarAhmedIOS** [comment]
> @EngOmarAhmedIOS @SSwiniarski Looks good. I just removed a couple of extra lines (there should never be multiple lines between content) and I decided to change the second example to a codebyte.

@caupolicandiaz is there anything I should do now

**Comment by EngOmarAhmedIOS** [comment]
Should I celebrate 🎉 this moment and consider my self as a contributor with Codeacadmey @caupolicandiaz 😁😁

**Comment by SSwiniarski** [comment]
@EngOmarAhmedIOS Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/python/built-in-functions/oct

image

**Pull request merged**